Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "review-web"
-version = "0.1.0"
+version = "0.1.0-alpha.1"
```

Comment:
This should be "0.2.0-alpha.2".

Reply:
I changed the version to "0.2.0-alpha.1"